src/Board.elm

```diff
@@ -0,0 +1,25 @@
+module Board exposing (..)
+
+import List exposing (repeat, take, drop, head)
+
+type alias Board = List String
+
+new : Int -> Board
+new size = 
+  repeat size ""
```

Comment:
👍  to `String` independency.
E.g: Is `"8t!ankl*/=qwet"` a valid value for a mark?